test/async.js

```diff
@@ -0,0 +1,288 @@
+// The API key and IDs used in this file belong to the official Ghost Inspector API testing account.
+const assert = require('assert').strict
+const fs = require('fs')
+const GhostInspector = require('../index')('ff586dcaaa9b781163dbae48a230ea1947f894ff')
```

Comment:
It'd be cool if this was loaded in from the environment somehow, I can live with the test IDs in the repo but it would be nice to have an alternative setup for this.